.env.cluster

```diff
@@ -69,3 +73,24 @@ LOGSTASH_MEMORY_LIMIT=8G
 ZOOKEEPER_MEMORY_LIMIT=4G
 KAFKA_MEMORY_LIMIT=8G
 KAFDROP_MEMORY_LIMIT=500M
+
+KC_FRONTEND_URL=https://keycloak.domain
+KEYCLOAK_ADMIN=admin
+KEYCLOAK_ADMIN_PASSWORD=dev_password_only
+KC_REALM_NAME=platform-realm
+KC_REPMGR_PRIMARY_HOST=keycloak-postgres-1
+KC_REPMGR_PARTNER_NODES=keycloak-postgres-1
+KC_REPMGR_PASSWORD=instant101
+KC_POSTGRES_REPLICA_SET=keycloak-postgres-1:5432
+KC_POSTGRES_CPU_LIMIT=0
+KC_POSTGRES_CPU_RESERVE=0.05
+KC_POSTGRES_MEMORY_LIMIT=3G
+KC_POSTGRES_MEMORY_RESERVE=500M
+KC_POSTGRESQL_PASSWORD=instant101
+KC_POSTGRESQL_USERNAME=admin
+KC_POSTGRESQL_DATABASE=keycloak
+KC_GRAFANA_CLIENT_ID=grafana-oauth
+KC_GRAFANA_CLIENT_SECRET=CV14QfwnpYFj1IH5dK5lScPNCYAIYP1c
+KC_GRAFANA_ROOT_URL=https://grafana.domain
+GF_SERVER_DOMAIN=grafana.domain
+GF_SERVER_ROOT_URL=https://grafana.domain
```

Comment:
Is all of these should be configured and relevant to consider when deploy ? I think we agreed to put here only the variables that should be overtided when deploy

Reply:
Yes, we need all of these to be set when doing a secure deploy. I will double-check the variables and see if we can remove any that are default, but most should be there.